src/SAWScript/Prover/Exporter.hs

```diff
@@ -477,12 +479,13 @@ writeCoqProp name notations skips path t =
      writeCoqTerm name notations skips path tm
 
 writeCoqCryptolModule ::
```

Comment:
Again, please add haddock for functions that you are modifying, even if it is just duplicating the textual description in `Interpreter.hs`

Reply:
Done in a997f3f61ae7b52747ee9af9931c2374f0004280! I did indeed basically just copy/paste the text from `Interpreter.hs`.